stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -415,13 +415,11 @@ function(_add_target_variant_link_flags)
   endif()
 
   if(NOT SWIFT_COMPILER_IS_MSVC_LIKE)
-    # FIXME: On Apple platforms, find_program needs to look for "ld64.lld"
-    find_program(LDLLD_PATH "ld.lld")
-    if((SWIFT_ENABLE_LLD_LINKER AND LDLLD_PATH AND NOT APPLE) OR
+    if((SWIFT_USE_LINKER STREQUAL "lld") OR
```

Comment:
I think that we should simplify this like the previous case in `cmake/modules/AddSwift.cmake`